mw-webapp/src/model/businessModel/JobDone.ts

```diff
@@ -25,7 +25,7 @@ export class JobDone {
    */
   public time: number;
 
-  constructor(jobDoneData: JobDone) {
+  constructor(jobDoneData: {uuid: string; description: string; timeUnit: TimeUnit; time: number}) {
```

Comment:
Why manual object type instead of class? Maybe i dont know something, please explain

Reply:
For don't create callback function in converter

Reply:
Don't understand how just types related to that but ok

Reply:
I see what you talked about, no problem then